package.json

```diff
@@ -98,7 +98,7 @@
     "stylelint-config-standard": "26.0.0",
     "ts-loader": "9.4.1",
     "ts-node": "10.9.1",
-    "ts-jest": "^26.5.6",
+    "ts-jest": "27.1.5",
```

Comment:
instead of updrading the version i was tried to remove the ts-jest configuration , that solves this error problem to me.